luigi/contrib/ftp.py

```diff
@@ -226,6 +227,11 @@ def __init__(
     ):
         if format is None:
             format = luigi.format.get_default_format()
+
+        # Allow to write unicode in file for retrocompatibility
+        if sys.version_info[:2] <= (2, 6):
```

Comment:
No it's really 2.6. The io.BufferedWriter accept unicode on python2.7. `io.BufferedWriter(io.FileIO('test.txt', 'w')).write(u'test')` works fine on 2.7. In fact, that's the reason that I didn't put the MixedUnicodeBytes format at the first place, I didn't think it was needed.